source/DSCResources/Resources/windows.UserRightsAssignment.ps1

```diff
@@ -24,39 +24,50 @@ $forestGroupTranslation = @{
     'Schema Admins'             = '{0}\Schema Admins'
 }
 
-# This requires a local forest and/or domain name to be injected to ensure a valid account name.
-$DomainName = PowerStig\Get-DomainName -DomainName $DomainName -Format NetbiosName
-$ForestName = PowerStig\Get-DomainName -ForestName $ForestName -Format NetbiosName
-
-#endregion Header
-
-foreach ($rule in $rules)
+if ($DomainName -and $ForestName)
 {
-    Write-Verbose $rule
-    $identitySplit = $rule.Identity -split ","
-    [System.Collections.ArrayList]  $identityList = @()
+    # This requires a local forest and/or domain name to be injected to ensure a valid account name.
+    $DomainName = PowerStig\Get-DomainName -DomainName $DomainName -Format NetbiosName
+    $ForestName = PowerStig\Get-DomainName -ForestName $ForestName -Format NetbiosName
+
+    #endregion Header
 
-    foreach ($identity in $identitySplit)
+    foreach ($rule in $rules)
     {
-        if ($domainGroupTranslation.Contains($identity))
-        {
-            [void] $identityList.Add($domainGroupTranslation.$identity -f $DomainName )
-        }
-        elseif ($forestGroupTranslation.Contains($identity))
+        Write-Verbose $rule
+        $identitySplit = $rule.Identity -split ","
+        [System.Collections.ArrayList]  $identityList = @()
```

Comment:
it appears there's an extra space before the "$".  It appears there's two spaces instead of one.